genshin/errors.py

```diff
@@ -166,15 +166,20 @@ class RedemptionClaimed(RedemptionException):
     -500004: VisitsTooFrequently,
     -502001: "User does not have this character.",
     -502002: "Calculator sync is not enabled.",
+    # diary
+    -400005: "Parameter error",
+    -501101: "This character has not reached Lv.10, please select another character.",
```

Comment:
If you're not gonna add any new information about the errors then don't there's no point adding them here.